manifests/install.pp

```diff
@@ -187,7 +208,7 @@
     timeout => 1800,
   }
   # Install gitlab with the appropriate package manager (rpm or dpkg)
-  package { 'gitlab':
+  package { "$::gitlab::install::gitlab_pkg":
```

Comment:
@spuder What version of gitlab are you trying to upgrade from?  It's almost like puppet is trying to upgrade the package, but because it was previously called gitlab and the new package is gitlab-ce it can't do the upgrade because there's no gitlab-ce installed to upgrade?